package.json

```diff
@@ -37,12 +34,12 @@
     "conventional-changelog-express": "^2.0.0",
     "conventional-changelog-jshint": "^2.0.0",
     "nyc": "^15.0.0",
-    "semantic-release": "^16.0.0",
+    "semantic-release": "^17.0.0",
     "sinon": "^8.0.0",
     "xo": "^0.25.0"
   },
   "engines": {
-    "node": ">=10.13"
+    "node": ">=10.18"
```

Comment:
Not sure if this was intentional, but the update to Ava forced an update to the engines range, which seems rather strange, considering all LTS releases have API and ABI stability, and 10.x.y is LTS